src/graphql/root/mutation/ln-invoice-payment-send.ts

```diff
@@ -29,6 +31,9 @@ const LnInvoicePaymentSendMutation = GT.Field({
     addAttributesToCurrentSpanAndPropagate(
       {
         [SemanticAttributes.ENDUSER_ID]: domainUser?.id,
+        [ENDUSER_PUBLICID]: domainUser?.id
```

Comment:
This is duplication of the line above right? `[SemanticAttributes.ENDUSER_ID]: domainUser?.id`

Reply:
It's part of the multi-line ternary on the two lines just below this